package.json

```diff
@@ -55,8 +56,9 @@
     "test": "jest --config jest.test.config.js",
     "test:watch": "jest --config jest.test.config.js --watch",
     "test:bundle": "jest --config jest.bundle.config.js",
-    "test:bundle:watch": "jest --config jest.bundle.config.js --watch",
-    "percy": "react-percy"
+    "test:percy": "percy exec -- yarn test:visual",
+    "test:visual": "jest --config jest.visual.config.js --maxWorkers=1",
```

Comment:
using multiple workers got errors from puppeteer about reusing requests... it seems to run relatively quickly like this though.

Reply:
For sequential runs, you can use `—runInBand`